packages/cli/workspace/loader/src/loadAPIWorkspace.ts

```diff
@@ -77,35 +78,34 @@ export async function loadSingleNamespaceAPIWorkspace({
                 fromOpenAPI: definition.schema.fromOpenAPI,
                 settings: {
```

Comment:
why can't we directly set settings, why do we have to copy all properties one by one?